tests_e2e/tests/agent_not_provisioned/disable_agent_provisioning.py

```diff
@@ -22,11 +22,14 @@
 from tests_e2e.tests.lib.update_arm_template import UpdateArmTemplate
 
 
-class DenyOutboundConnections(UpdateArmTemplate):
```

Comment:
yup, the result of copy-n-paste programming :(

Reply:
the incorrect name did not affect functionality, though, since we load the class based on type (inhertis from UpdateArmTemplate) and the name is not used